src/main/java/solvd/laba/ermakovich/hu/event/kafka/KafkaDoctorHandler.java

```diff
@@ -0,0 +1,63 @@
+package solvd.laba.ermakovich.hu.event.kafka;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import org.springframework.stereotype.Service;
+import reactor.core.publisher.Mono;
+import solvd.laba.ermakovich.hu.domain.Doctor;
+import solvd.laba.ermakovich.hu.domain.exception.IllegalOperationException;
+import solvd.laba.ermakovich.hu.event.elastic.ElasticDoctorEventService;
+import solvd.laba.ermakovich.hu.event.integration.CreateElasticDoctor;
+import solvd.laba.ermakovich.hu.event.integration.DeleteElasticDoctor;
+import solvd.laba.ermakovich.hu.event.integration.IntegrationEvent;
+
+/**
+ * @author Ermakovich Kseniya
+ */
+@Service
+@RequiredArgsConstructor
+public final class KafkaDoctorHandler implements DoctorService {
```

Comment:
@kermakovich What is the involvement of the kafka here?
maybe I miss something..

Reply:
@h1alexbel  It is a service, that handles kafka consumer events. That is why I've added 'kafka' here.  I think it's better to change interface name. For example, KafkaDoctorService or KafkaDoctorHandler